packages/bot-web-ui/src/components/quick-strategy/form-wrappers/mobile-form-wrapper.tsx

```diff
@@ -73,7 +73,7 @@ const MobileFormWrapper: React.FC<TMobileFormWrapper> = observer(({ children })
                         <FormTabs
                             active_tab={active_tab}
                             onChange={handleTabChange}
-                            description={strategy?.long_description}
+                            description={strategy?.long_description || strategy?.description}
```

Comment:
@maryia-matskevich-deriv can we keep only 1 out of long_description and description? since we are showing everything only in the `Learn more` tab

Reply:
done✅

Reply:
done✅